CDS Maps/Individual Maps/CDS-to-FHIR/nNGM-CDS-to-FHIR-Operation.map

```diff
@@ -15,7 +10,7 @@ uses "http://hl7.org/fhir/StructureDefinition/Patient" as target
 group TransformBundleOperationTherapie(source operations: BackboneElement, target bundle: Bundle, target composition: Composition, target section: BackboneElement)
 {
     operations then CreateProcedureOperationTherapie(operations, bundle, composition, section);
-    operations then CreatePatientOperationTherapie(operations, bundle, section);
+    // operations then CreatePatientOperationTherapie(operations, bundle, section);
```

Comment:
Everything regarding the patient can be deleted

Reply:
410